ui/app/templates/components/database-connection.hbs

```diff
@@ -107,7 +107,7 @@
                 <div class="columns is-desktop is-multiline">
                   {{#each fields as |attr|}}
                     {{#if
-                      (contains attr.name (array "max_open_connections" "max_idle_connections" "max_connection_lifetime"))
```

Comment:
`contains` is fully deprecated in the latest version of `ember-composable-helpers`